x/dualstaking/keeper/grpc_query_provider_delegators.go

```diff
@@ -0,0 +1,43 @@
+package keeper
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/utils"
+	"github.com/lavanet/lava/x/dualstaking/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (k Keeper) ProviderDelegators(goCtx context.Context, req *types.QueryProviderDelegatorsRequest) (*types.QueryProviderDelegatorsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
```

Comment:
need to validate the input `provider` and throw an error if invalid.
(otherwise this would return valid reply (with empty delegation list) for this provider.

Reply:
done. see https://github.com/lavanet/lava/commit/991a0e74722061319cbef1cd3199f4f893b87702